### DISCUSSION TITLE
Opeining files in evil mode

### DISCUSSION THREAD

**Discussion by alsoijw** [syl20bnr/spacemacs#15060] [Q&A]
I have a project with the following structure
```
.
lib/
lib/a
lib/b
test/
test/a
test/b
```
I know the structure of the project, so I can open different files.

```nvim lib/a
:e test/a
:e lib/b
```

However, in spacemacs, the file is opened relative to the most recent folder in which the last file is opened.

```emacs lib/a
:e ../test/a
:e ../lib/b
```

Since it is quite difficult to remember which file was opened last, you have to constantly climb to the root level of the project and open the file from there. How to bring the behavior to the way it happens in vim?

**Comment by dalanicolai** [comment]
I don't know about others, but personally I generally use `SPC f r` for this, and I guess most users prefer to use `SPC p f` (I don't work much with projects (yet), but in Doom `SPC p f` is regarded so 'essential' that it is mapped on `SPC SPC`).

I do not have the answer to your exact question. If it is very important then we could probably find it out (although I guess you could also/better ask the `evil` developers).

Did you try to ask this on [gitter](https://gitter.im/syl20bnr/spacemacs) first b.t.w.? I think that not many users keep an eye on the discussions section.
